src/move/data/preprocessing.py

```diff
@@ -74,7 +74,9 @@ def scale(x: np.ndarray) -> tuple[FloatArray, BoolArray]:
         Tuple containing (1) scaled output and (2) a 1D mask marking columns
         (i.e., features) without zero variance
     """
-    logx = np.log2(x + 1)
+    # CHANGES: Allow for negative values, so only standardization
```

Comment:
What was the reason for that change?

Reply:
@ri-heme What is your take on negative data? Should the log-normalization be optional? Or should it always be applied?